src/app/initialisation/syncPoolService.js

```diff
@@ -40,12 +40,11 @@ function SyncPoolService($q, mediator, syncService) {
 
     syncManagers = {};
 
-    if (profileData && profileData.id) {
-      var filter = {
-        key: 'assignee',
-        value: profileData.id
-      };
-    }
+    var filter = {
+      key: 'assignee',
+      value: profileData.id,
```

Comment:
I agree that this value is unused and then swapped by actual id in the cloud but putting token here will be little bit misleading for someone who will read that code - expectance may be that assignee in database will have token value. When having session token as separate field we can connect it with any other query like:
status: completed and assigned to this user (using token)